src/elements/ProjectJobs/projectJobs.utils.js

```diff
@@ -15,6 +16,12 @@ export const getJobsStatistics = (jobs, match, scheduledJobs, workflows) => {
       0
     )
   }
+  if (workflows.data) {
+    workflowsRunning = workflows.data.reduce(
+      (prev, curr) => (curr.status === 'Running' ? (prev += 1) : prev),
+      0
+    )
```

Comment:
I know we're doing this `reduce` approach in many other places, but, now when I think about it, it may be simpler to use `filter` and then `.length` on the result:
```suggestion
    workflowsRunning = workflows.data.filter(workflow => workflow === 'Running').length
```